extensions/distributed-tracing.md

```diff
@@ -1,55 +1,43 @@
 # Distributed Tracing extension
 
 This extension embeds context from
-[Distributed Tracing](https://w3c.github.io/trace-context/) so that distributed
-systems can include traces that span an event-driven system. This extension is
-meant to contain historical data of the parent trace, in order to diagnose
-eventual failures of the system through tracing platforms like Jaeger, Zipkin,
-etc.
+[W3C TraceContext](https://www.w3.org/TR/trace-context/) into a CloudEvent.
```

Comment:
Maybe add a link to https://cloudevents.io/?

Reply:
Hum it's already the CloudEvents specification, so I guess it's clear enough? 😅